microsoft-edge/devtools-guide-chromium/whats-new/2023/06/devtools-114.md

```diff
@@ -0,0 +1,193 @@
+> * [Microsoft Edge | Bringing WebView2 to Microsoft Teams and beyond](https://www.youtube.com/watch?v=s3tDUvaoCP4&list=PL4z1-7pjJU6zJT3PBQ4mTbNg2wtX7Lt52)
+> * [Building Chat Plugins for Microsoft Bing and Edge](https://www.youtube.com/watch?v=Q-5M7EYjl6U&list=PL4z1-7pjJU6zJT3PBQ4mTbNg2wtX7Lt52)
+
+
+<!-- ====================================================================== -->
+## Aggregate Selector Stats in the Performance tool 
+
+<!-- Subtitle: Instead of selecting individual Recalculate Style events when recording a profile, the Selector Stats tab now aggregates data across all Recalculate Style events-->
+
+<!-- Reviewer: Gavin Xu -->
+
+In [Microsoft Edge 109](../01/devtools-109.md#debug-long-running-recalculate-style-events-with-new-selector-stats), we added the **Selector Stats** feature to the **Performance** tool. You can use the data from **Selector Stats** to understand which CSS selectors are taking the most time during **Recalculate Style** events in the **Performance** tool and contributing to slow performance.
+
+Now in Microsoft Edge 114, you no longer need to select an individual **Recalculate Style** event to see the selectors that were recalculated during that event. Rather, the **Selector Stats** tab in the bottom pane of the **Performance** tool automatically aggregates the data across all of the **Recalculate Style** events in the recorded profile. As you zoom into specific parts of the profile, the **Selector Stats** tab updates to only show data from the portion of the profile that you are currently analyzing. 
+
+Additionally, a new column called **Style sheet** has been added to the **Selector Stats** tab and contains links for each selector back to the stylesheet where the selectors are defined. 
+
+![Selector Stats are now aggregated across Recalculate Style events within the currently displayed section of the recorded profile](./devtools-114-images/aggregate-selector-stats.png)
+
+<!-- todo
+1. Open Devtools and navigate to the performance tool
+2. Hit the capture settings button
+3. Check the box at the top that says "Enable advanced rendering instrumentation (slow)"
+4. Click on a purple "Recalculate style" box
+5. Click on the Selector stats button on the bar below
+-->
+
+Thank you for using the **Selector Stats** feature and sharing your feedback with us in [GitHub Issue #98: \[Feedback\] Selector Performance Tracing Explainer](https://github.com/MicrosoftEdge/DevTools/issues/98)!
+
+See also:
+* [Analyze selector performance during Recalculate Style events](../../../evaluate-performance/selector-stats.md)
+* [GitHub Issue #98: \[Feedback\] Selector Performance Tracing Explainer](https://github.com/MicrosoftEdge/DevTools/issues/98)
+
+<!-- ====================================================================== -->
+## The Issues tool warns you when CSS properties trigger reflow 
```

Comment:
`reflow` is kind of overloaded to group Paint, Composite, Layout or a combination of them this is fine for internal use but for external use I would the more scoped term `Layout`.

Reply:
I will apply this comment to [PR 2618](https://github.com/MicrosoftDocs/edge-developer/pull/2618)

Reply:
done locally for 2618

Reply:
3
@jenna-sasson - this comment thread can be closed.

gh rendered section in other PR's branch:
https://github.com/mikehoffms/edge-developer/blob/user/mikehoffms/whats-new-114/microsoft-edge/devtools-guide-chromium/whats-new/2023/06/devtools-114.md#the-issues-tool-and-styles-pane-warn-about-css-properties-that-trigger-layout